concourse/pipelines/bosh-cli/bosh-cli.yml

```diff
@@ -0,0 +1,20 @@
+---
+platform: linux
+
+image: docker:///governmentpaas/bosh-cli
+
+inputs:
+  - name: cf-manifest
+  - name: bosh-secrets
+run:
+  path: sh
+  args:
+  - -c
+  - -e
+  - |
+    bosh_password=$(awk '/bosh_admin_password/ { print $2 }' bosh-secrets/bosh-secrets.yml)
+    bosh -t 10.0.0.6 login admin "${bosh_password}"
+    bosh target 10.0.0.6
+    uuid=$(bosh status --uuid)
+    sed -e "s/^director_uuid:.*$/director_uuid: ${uuid}/" cf-manifest/cf-manifest.yml > cf-manifest-with-uuid.yml
```

Comment:
Are these two lines necessary? Doesn't `deploy-cloudfoundry/deploy`, which this pipeline takes inputs from, do this and store the result already?

- https://github.com/alphagov/paas-cf/blob/master/concourse/pipelines/deploy-cloudfoundry.yml#L498